src/torchmetrics/text/infolm.py

```diff
@@ -148,6 +162,7 @@ def __init__(
 
         self.tokenizer, self.model = _load_tokenizer_and_model(model_name_or_path, device)
         self.information_measure_cls = _InformationMeasure(information_measure, alpha, beta)
+        self.higher_is_better = _Information_Measure_Higher_Is_Better[information_measure]
```

Comment:
also this need to be a property overwriting the constant attribute

Reply:
I looked at what @property was being used for and seemed like it would more coincide with setting it like this since its tied to the measure (so wont need to be updated/changed), but sure, will make it a property